src/main.ts

```diff
@@ -40,10 +38,16 @@ async function main() {
             type: 'string',
             describe: 'The file to translate (leave out for stdin)',
           })
+          .option('language', {
+            type: 'string',
+            describe: 'Language ID to transliterate to',
+            choices: Array.from(new Set(Object.values(TargetLanguage))),
```

Comment:
deduplicate choices using set here, because there are two `python` in the enum

https://github.com/aws/jsii-rosetta/blob/acdefc136e96043f17c6585e6d2bf0ae6be572a1/src/languages/target-language.ts#L3-L10